src/components/Nav/index.js

```diff
@@ -323,7 +323,8 @@ const Nav = ({ handleThemeChange, isDarkTheme, path }) => {
     }
   }
 
-  const shouldShowSubNav = path.includes("/developers/")
+  const shouldShowSubNav =
```

Comment:
Is the thinking here to have a community subnav on the community pages?

I'm open to this but would probably default to just adding these subpages to the Community dropdown nav.